library/Init/Lean/Elaborator/Basic.lean

```diff
@@ -386,12 +375,15 @@ match fileName with
 
 def testFrontend (input : String) (fileName : Option String := none) : IO (Environment × MessageLog) :=
 do env ← mkEmptyEnvironment;
-   baseDir ← toBaseDir fileName;
+   baseMod ← match fileName with
+   -- TODO: (try to) resolve current working directory to support relative imports with --stdin
```

Comment:
I am happy to remove support for relative imports.

Reply:
Yeah, I thought of that simplification too late and was surprised to find that relative imports are used exactly twice in mathlib (so probably an oversight). I will remove them.